pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_deleting-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,19 @@
+[id="deleting-a-module-in-pantheon_{context}"]
+
+= Deleting a module in {ProductShortName}
+
+Deleting a module in {ProductShortName}.
+
+.Prerequisites
+
+//* You are assigned a publisher or admin role in {ProductShortName}.
+* You are logged in.
+
+.Procedure
+
+. Navigate to the link:{LinkToSearchPage}[Search page].
+. Select the modules you want to delete.
```

Comment:
Just a nit-pick, the module heading is about a module, but this step is about modules. Probably doesn't warrant an edit!